doc/reports/prelude.rst

```diff
@@ -0,0 +1,82 @@
+| 0.2        |    September 14, 2020 | Draft reviewed by the Testing WG members. Assigned maintainers of the each section |
++------------+-----------------------+------------------------------------------------------------------------------------+
+| 0.3        |    September 21, 2020 | Draft reviewed by the Testing WG members before the release v.2.4.0                |
++------------+-----------------------+------------------------------------------------------------------------------------+
+| 0.4        |    September 28, 2020 | Draft reviewed by the Testing WG members after the release v.2.4.0                 |
++------------+-----------------------+------------------------------------------------------------------------------------+
+| 0.5        |    December 07, 2020  | Refactored. Move details to the appendix and wiki. Some rewording. Added           |
+|            |                       | Cross-reference plugin in add-ons that cover automatic enumeration                 |
+|            |                       | andcross-referencing for tables and figure                                         |
++------------+-----------------------+------------------------------------------------------------------------------------+
+
+
+`Zephyr Testing Working Group
+<mailto:testing-wg@lists.zephyrproject.org>`_
+
+
+Goals
+=====
+
+This report is created to ease the evaluation of the current Zephyr release so that stakeholders can understand the project status and the quality of the release. To achieve this the report:
+
+    #. Provides the key testing results of the released version.
+    #. Gives an overview of the Zephyr RTOS health
+    #. Informs about the open issues.
+
+Intended audience
+=================
+
+The report can be used by anyone interested in the health of Zephyr, like TSC members, Zephyr community developers, and stakeholders considering equipping their platforms with the Zephyr RTOS. 
+
+
+Legal
+=====
+
+Testing WG community does not make any representations or warranties as to the accuracy of the information, quality, reliability, functionality or compatibility. Names and brands in that report may be claimed as the property of others. Our Testing WG community members don't receive any commercial benefit from the board’s manufacturers. We obtain test results using boards which we have on hands during our daily work. If you want your platform results to be added here, you can send your test results by contacting the Testing WG community using that email Testing community will not verify data provided by you and doesn’t make any claims about the reliability and correctness of it. We understand that performance and all test results vary depending on system configuration. Provided data below can’not be used in any marketing activities or advertisements. No certification issued for all results which you can view in that document. 

```

Comment:
"Provided data below can'not be....."

" ' " should not removed.

Reply:
ok